js/src/pages/settings/index.js

```diff
@@ -35,6 +36,8 @@ const Settings = () => {
 
 	const { google } = useGoogleAccount();
 	const isReconnectGooglePage = subpath === subpaths.reconnectGoogleAccount;
+	const { hasGoogleMCConnection } = useGoogleMCAccount();
+	const hasMC = hasGoogleMCConnection || glaData?.mcSetupComplete;
```

Comment:
`hasGoogleMCConnection` should be enough and we don't need to check for `glaData?.mcSetupComplete`